pkgs/halo2-wasm-ext/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@anonklub/halo2-wasm-ext",
+  "version": "1.1.3",
+  "author": "0xisk <0xisk@proton.me>",
+  "description": "A Rust crate for overriding axiom halo2-wasm package.",
+  "keywords": [
+    "wasm",
+    "halo2wasm",
+    "rust"
+  ],
+  "license": "MIT",
+  "repository": "https://github.com/anonklub/anonklub",
+  "homepage": "https://github.com/anonklub/anonklub",
+  "bugs": "https://github.com/anonklub/anonklub/issues",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist/**/*.{js,ts,wasm}",
+    "README.md",
+    "CHANGELOG.md"
+  ],
+  "module": "dist/index.js",
+  "sideEffects": [
+    "dist/index.js"
+  ],
+  "scripts": {
+    "clean": "rm -rf dist",
+    "check-wasm-pack": "command -v wasm-pack >/dev/null 2>&1 || echo \"wasm-pack is not installed\"\n",
+    "format.cargo": "cargo fmt -- --check",
+    "format.cargo.fix": "cargo fmt",
+    "lint.cargo": "cargo clippy",
+    "lint.cargo.fix": "cargo clippy --fix",
+    "prepack": "pnpm compile"
```

Comment:
there is no `compile` script in this package

Reply:
do we even need to compile with `wasm-pack`?
Is `halo2-wasm-ext` supposed to be packaged into wasm too?

Reply:
Actually not, I will remove it.

Reply:
Also we don't need the whole package.json.

Reply:
great